equivalences.tex

```diff
@@ -87,8 +87,8 @@ \section{Quasi-inverses}
   Let $g:\prd{x:A} \brck{a=x}$ be as given by~\ref{item:autohtpy2}.  First we
   observe that each type $\id[A]xy$ is a set.  For since being a set is a mere
   proposition, we may apply the induction principle of propositional truncation, and assume that $g(x)=\bproj
-  p$ and $g(y)=\bproj q$ for $p:a=x$ and $q:a=y$.  In this case, composing with
-  $p$ and $\opp{q}$ yields an equivalence $\eqv{(x=y)}{(a=a)}$.  But $(a=a)$ is
+  p$ and $g(y)=\bproj{p'}$ for $p:a=x$ and $p':a=y$.  In this case, composing with
```

Comment:
The variable q occurs a lot of times in the statement of Lemma 4.1.2 and in its proof. It seems to me that the first three occurrences of q in the proof should be changed to a name of variable which doesn't appear elsewhere in the proof. Note for instance that we have q:a=a in the statement, and q:a=y at the beginning of the proof.